src/drivers/imu/adis16448/adis16448.cpp

```diff
@@ -224,6 +226,7 @@ class ADIS16448 : public device::SPI
 
 	uint16_t			_product;	/** product code */
 
+	work_s			_work{};
```

Comment:
why the empty initialization list?

Reply:
hacky way of initializing everything to zero - seen in other parts of nuttx/px4 drivers that use the work queue